### PR TITLE
docs: Tweaking steps for garbage collection

### PR DESCRIPTION
**Pull request by karenarialin**
**What type of PR is this?**
/kind docs

**What this PR does / Why we need it**:
As per troubleshooting session with yuryu, a couple tweaks are needed for the garbage collection section.
-Adding Datastore index.yaml
-Fix docker run command

**Which issue(s) this PR fixes**:
N/A

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/deploying.md [hunk 1]
```diff
@@ -156,6 +156,34 @@ and cannot be undone. Google Cloud Platform currently allows only one Datastore
 database per project, so you would need to create a new project to change the
 database location.
 
+Next, create the following `index.yaml` file:
+
+```bash
+cat > index.yaml << 'EOF'
+indexes:
```
Comment (yuryu) [inline docs/deploying.md:163]: Can we move the content of the file to a separate file, e.g. 
deploy/datastore/index.yaml
and ask users to just use the file?

```
gcloud datastore indexes create deploy/datastore/index.yaml
```

should read from the file.

**Review by yuryu — APPROVED** [review]
lgtm with comments

File: docs/deploying.md (hunk 1 above)
Comment (karenarialin) [reply docs/deploying.md]: Done.

**Review by yuryu — APPROVED** [review]
lgtm

**Pull request merged**